tests/ifs/sandstone_ifs.h

```diff
@@ -16,6 +16,8 @@
 #define IFS_SW_PARTIAL_COMPLETION               0xFE
 
 typedef struct {
+    const char *sys_dir;
+    const char *sys_path;
```

Comment:
Shouldn't be needed to keep both, as the base is constant.

Reply:
right, I was just being lazy. Fixed now as well. :)